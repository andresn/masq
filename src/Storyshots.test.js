import initStoryshots from '@storybook/addon-storyshots'

test('adds 1 + 2 to equal 3', () => {
  expect(1 + 2).toBe(3)
})

initStoryshots()
