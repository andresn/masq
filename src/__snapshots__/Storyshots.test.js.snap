// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Avatar with upload and with image 1`] = `
<div>
  <input
    accept=".jpg, .jpeg, .png"
    name="avatar"
    onChange={undefined}
    style={
      Object {
        "display": "none",
      }
    }
    type="file"
  />
  <div
    className="Avatar upload"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url(https://randomuser.me/api/portraits/women/79.jpg)",
      }
    }
  >
    <svg
      className="camera"
      color="white"
      height={33}
      viewBox="0 0 24 20"
      width={40}
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M21 3h-3.5L15.8.4c-.2-.2-.5-.4-.8-.4H9c-.3 0-.6.2-.8.4L6.5 3H3C1.3 3 0 4.3 0 6v11c0 1.7 1.3 3 3 3h18c1.7 0 3-1.3 3-3V6c0-1.7-1.3-3-3-3zm1 14c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1V6c0-.6.4-1 1-1h4c.3 0 .6-.2.8-.4L9.5 2h4.9l1.7 2.6c.3.2.6.4.9.4h4c.6 0 1 .4 1 1v11zM12 6c-2.8 0-5 2.2-5 5s2.2 5 5 5 5-2.2 5-5-2.2-5-5-5zm0 8c-1.7 0-3-1.3-3-3s1.3-3 3-3 3 1.3 3 3-1.3 3-3 3z"
          id="camera-svg"
        />
      </defs>
      <use
        fill="currentColor"
        fillRule="nonzero"
        xlinkHref="#camera-svg"
      />
    </svg>
  </div>
</div>
`;

exports[`Storyshots Avatar with upload and without image 1`] = `
<div>
  <input
    accept=".jpg, .jpeg, .png"
    name="avatar"
    onChange={undefined}
    style={
      Object {
        "display": "none",
      }
    }
    type="file"
  />
  <svg
    className="Avatar upload"
    color="var(--grey2-color)"
    height={120}
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url(undefined)",
      }
    }
    viewBox="0 0 120 120"
    width={120}
  >
    <g
      fill="none"
      fillRule="evenodd"
    >
      <circle
        cx={60}
        cy={60}
        fill="#fff"
        r={60}
      />
      <circle
        cx={60}
        cy={60}
        r={59}
        stroke="currentColor"
        strokeWidth={2}
      />
      <path
        d="M76.8 46.933h-6.533l-3.174-4.853c-.373-.373-.933-.747-1.493-.747H54.4c-.56 0-1.12.374-1.493.747l-3.174 4.853H43.2c-3.173 0-5.6 2.427-5.6 5.6v20.534c0 3.173 2.427 5.6 5.6 5.6h33.6c3.173 0 5.6-2.427 5.6-5.6V52.533c0-3.173-2.427-5.6-5.6-5.6zm1.867 26.134c0 1.12-.747 1.866-1.867 1.866H43.2c-1.12 0-1.867-.746-1.867-1.866V52.533c0-1.12.747-1.866 1.867-1.866h7.467c.56 0 1.12-.374 1.493-.747l3.173-4.853h9.147l3.173 4.853c.56.373 1.12.747 1.68.747H76.8c1.12 0 1.867.746 1.867 1.866zM60 52.533c-5.227 0-9.333 4.107-9.333 9.334 0 5.226 4.106 9.333 9.333 9.333s9.333-4.107 9.333-9.333c0-5.227-4.106-9.334-9.333-9.334zm0 14.934c-3.173 0-5.6-2.427-5.6-5.6 0-3.174 2.427-5.6 5.6-5.6s5.6 2.426 5.6 5.6c0 3.173-2.427 5.6-5.6 5.6z"
        fill="currentColor"
        fillRule="nonzero"
      />
    </g>
  </svg>
</div>
`;

exports[`Storyshots Avatar without upload and with image 1`] = `
<div
  className="Avatar"
  style={
    Object {
      "backgroundImage": "url(https://randomuser.me/api/portraits/women/79.jpg)",
    }
  }
/>
`;

exports[`Storyshots Button primary with a label 1`] = `
<button
  className="Button primary"
  onClick={[Function]}
>
  somebutton
</button>
`;

exports[`Storyshots Button secondary with a label 1`] = `
<button
  className="Button secondary"
  onClick={[Function]}
>
  somebutton
</button>
`;

exports[`Storyshots Card disabled 1`] = `
<div
  className="Card"
>
  <div
    className="flex"
  >
    <div
      className="content"
    >
      <div
        className="CardHeader"
      >
        <div
          className="line"
          style={
            Object {
              "backgroundColor": "var(--grey2-color)",
            }
          }
        />
        <div
          className="SwitchButton"
        >
          <label
            style={
              Object {
                "color": null,
              }
            }
          >
            DISABLED
          </label>
          <div
            className=""
            id="switch-container"
            onClick={[Function]}
          >
            <div
              className=""
              id="background"
              style={Object {}}
            />
            <div
              className=""
              id="cursor"
              style={Object {}}
            />
          </div>
        </div>
      </div>
      <h1>
        sometitle
      </h1>
    </div>
  </div>
</div>
`;

exports[`Storyshots Card enabled 1`] = `
<div
  className="Card"
>
  <div
    className="flex"
  >
    <div
      className="content"
    >
      <div
        className="CardHeader"
      >
        <div
          className="line"
          style={
            Object {
              "backgroundColor": "blue",
            }
          }
        />
        <div
          className="SwitchButton"
        >
          <label
            style={
              Object {
                "color": "blue",
              }
            }
          >
            ENABLED
          </label>
          <div
            className=""
            id="switch-container"
            onClick={[Function]}
          >
            <div
              className=""
              id="background"
              style={
                Object {
                  "backgroundColor": "blue",
                }
              }
            />
            <div
              className=""
              id="cursor"
              style={
                Object {
                  "transform": "translate3d(26px, 0, 0)",
                }
              }
            />
          </div>
        </div>
      </div>
      <h1>
        sometitle
      </h1>
    </div>
  </div>
</div>
`;

exports[`Storyshots Card enabled with a description 1`] = `
<div
  className="Card"
>
  <div
    className="flex"
  >
    <div
      className="content"
    >
      <div
        className="CardHeader"
      >
        <div
          className="line"
          style={
            Object {
              "backgroundColor": "blue",
            }
          }
        />
        <div
          className="SwitchButton"
        >
          <label
            style={
              Object {
                "color": "blue",
              }
            }
          >
            ENABLED
          </label>
          <div
            className=""
            id="switch-container"
            onClick={[Function]}
          >
            <div
              className=""
              id="background"
              style={
                Object {
                  "backgroundColor": "blue",
                }
              }
            />
            <div
              className=""
              id="cursor"
              style={
                Object {
                  "transform": "translate3d(26px, 0, 0)",
                }
              }
            />
          </div>
        </div>
      </div>
      <h1>
        sometitle
      </h1>
      <p
        className="description"
      >
        some description
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Card enabled with a logo 1`] = `
<div
  className="Card"
>
  <div
    className="flex"
  >
    <div
      className="content"
    >
      <div
        className="CardHeader"
      >
        <div
          className="line"
          style={
            Object {
              "backgroundColor": "blue",
            }
          }
        />
        <div
          className="SwitchButton"
        >
          <label
            style={
              Object {
                "color": "blue",
              }
            }
          >
            ENABLED
          </label>
          <div
            className=""
            id="switch-container"
            onClick={[Function]}
          >
            <div
              className=""
              id="background"
              style={
                Object {
                  "backgroundColor": "blue",
                }
              }
            />
            <div
              className=""
              id="cursor"
              style={
                Object {
                  "transform": "translate3d(26px, 0, 0)",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <div
          id="logo"
          style={
            Object {
              "backgroundImage": "url(https://upload.wikimedia.org/wikipedia/commons/f/fd/Qwant-v3.png)",
              "backgroundPosition": "top center",
              "backgroundSize": "cover",
              "height": "16px",
              "marginRight": "4px",
              "width": "16px",
            }
          }
        />
        <h1>
          sometitle
        </h1>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Card enabled with an image 1`] = `
<div
  className="Card"
>
  <div
    className="image"
    style={
      Object {
        "backgroundImage": "url(https://images.pexels.com/photos/63703/pexels-photo-63703.jpeg?w=1260&h=750&auto=compress&cs=tinysrgb)",
      }
    }
  />
  <div
    className="flex"
  >
    <div
      className="content"
    >
      <div
        className="CardHeader"
      >
        <div
          className="line"
          style={
            Object {
              "backgroundColor": "blue",
            }
          }
        />
        <div
          className="SwitchButton"
        >
          <label
            style={
              Object {
                "color": "blue",
              }
            }
          >
            ENABLED
          </label>
          <div
            className=""
            id="switch-container"
            onClick={[Function]}
          >
            <div
              className=""
              id="background"
              style={
                Object {
                  "backgroundColor": "blue",
                }
              }
            />
            <div
              className=""
              id="cursor"
              style={
                Object {
                  "transform": "translate3d(26px, 0, 0)",
                }
              }
            />
          </div>
        </div>
      </div>
      <h1>
        sometitle
      </h1>
    </div>
  </div>
</div>
`;

exports[`Storyshots CardHeader disabled 1`] = `
<div
  className="CardHeader"
>
  <div
    className="line"
    style={
      Object {
        "backgroundColor": "var(--grey2-color)",
      }
    }
  />
  <div
    className="SwitchButton"
  >
    <label
      style={
        Object {
          "color": null,
        }
      }
    >
      DISABLED
    </label>
    <div
      className=""
      id="switch-container"
      onClick={[Function]}
    >
      <div
        className=""
        id="background"
        style={Object {}}
      />
      <div
        className=""
        id="cursor"
        style={Object {}}
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots CardHeader enabled 1`] = `
<div
  className="CardHeader"
>
  <div
    className="line"
    style={
      Object {
        "backgroundColor": "blue",
      }
    }
  />
  <div
    className="SwitchButton"
  >
    <label
      style={
        Object {
          "color": "blue",
        }
      }
    >
      ENABLED
    </label>
    <div
      className=""
      id="switch-container"
      onClick={[Function]}
    >
      <div
        className=""
        id="background"
        style={
          Object {
            "backgroundColor": "blue",
          }
        }
      />
      <div
        className=""
        id="cursor"
        style={
          Object {
            "transform": "translate3d(26px, 0, 0)",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots CircleIndicator with color blue 1`] = `
<div
  className="CircleIndicator"
  style={
    Object {
      "border": "solid 2px blue",
    }
  }
/>
`;

exports[`Storyshots CircleIndicator with color red 1`] = `
<div
  className="CircleIndicator"
  style={
    Object {
      "border": "solid 2px red",
    }
  }
/>
`;

exports[`Storyshots Header default header 1`] = `
<div>
  <div
    id="invisible"
    style={
      Object {
        "height": 64,
      }
    }
  />
  <div
    className="Header"
  >
    <div
      id="top-section"
    >
      <div
        id="title-section"
      >
        <p
          id="title"
        >
          Masq
        </p>
        <p>
           by Qwant
        </p>
      </div>
      <a
        href="login"
        onClick={[Function]}
      >
        <svg
          className="home"
          height={22}
          onClick={undefined}
          viewBox="0 0 20 22"
          width={20}
          xmlnsXlink="http://www.w3.org/1999/xlink"
        >
          <defs>
            <path
              d="M19.6 7.217l-9-6.992c-.4-.3-.9-.3-1.2 0l-9 6.992c-.3.2-.4.5-.4.799v10.987C0 20.701 1.3 22 3 22h14c1.7 0 3-1.299 3-2.997V8.016c0-.3-.1-.6-.4-.8zM12 20.002H8v-7.99h4v7.99zm6-.999c0 .6-.4 1-1 1h-3v-8.99c0-.6-.4-1-1-1H7c-.6 0-1 .4-1 1v8.99H3c-.6 0-1-.4-1-1V8.515l8-6.193 8 6.193v10.488z"
              id="home-svg"
            />
          </defs>
          <use
            fill="currentColor"
            fillRule="nonzero"
            xlinkHref="#home-svg"
          />
        </svg>
      </a>
    </div>
  </div>
</div>
`;

exports[`Storyshots Header with a user logged in 1`] = `
<div>
  <div
    id="invisible"
    style={
      Object {
        "height": 64,
      }
    }
  />
  <div
    className="Header"
  >
    <div
      id="top-section"
    >
      <div
        id="avatar-section"
      >
        <div
          className="Avatar"
          style={
            Object {
              "backgroundImage": "url(https://randomuser.me/api/portraits/women/79.jpg)",
            }
          }
        />
        <span
          id="avatar-text"
          style={
            Object {
              "fontSize": "14px",
            }
          }
        >
          some username
        </span>
        <svg
          className="caret"
          height={10}
          viewBox="0 0 14 10"
          width={14}
          xmlnsXlink="http://www.w3.org/1999/xlink"
        >
          <defs>
            <path
              d="M13.7 1.7l-6 6c-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3l-6-6C-.1 1.3-.1.7.3.3c.4-.4 1-.4 1.4 0L7 5.6 12.3.3c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4z"
              id="chevron-svg"
            />
          </defs>
          <use
            fill="currentColor"
            fillRule="nonzero"
            xlinkHref="#chevron-svg"
          />
        </svg>
      </div>
      <a
        href="login"
        onClick={[Function]}
      >
        <svg
          className="home"
          height={22}
          onClick={undefined}
          viewBox="0 0 20 22"
          width={20}
          xmlnsXlink="http://www.w3.org/1999/xlink"
        >
          <defs>
            <path
              d="M19.6 7.217l-9-6.992c-.4-.3-.9-.3-1.2 0l-9 6.992c-.3.2-.4.5-.4.799v10.987C0 20.701 1.3 22 3 22h14c1.7 0 3-1.299 3-2.997V8.016c0-.3-.1-.6-.4-.8zM12 20.002H8v-7.99h4v7.99zm6-.999c0 .6-.4 1-1 1h-3v-8.99c0-.6-.4-1-1-1H7c-.6 0-1 .4-1 1v8.99H3c-.6 0-1-.4-1-1V8.515l8-6.193 8 6.193v10.488z"
              id="home-svg"
            />
          </defs>
          <use
            fill="currentColor"
            fillRule="nonzero"
            xlinkHref="#home-svg"
          />
        </svg>
      </a>
    </div>
  </div>
</div>
`;

exports[`Storyshots Header with shadow 1`] = `
<div>
  <div
    id="invisible"
    style={
      Object {
        "height": 64,
      }
    }
  />
  <div
    className="Header shadow"
  >
    <div
      id="top-section"
    >
      <div
        id="title-section"
      >
        <p
          id="title"
        >
          Masq
        </p>
        <p>
           by Qwant
        </p>
      </div>
      <a
        href="login"
        onClick={[Function]}
      >
        <svg
          className="home"
          height={22}
          onClick={undefined}
          viewBox="0 0 20 22"
          width={20}
          xmlnsXlink="http://www.w3.org/1999/xlink"
        >
          <defs>
            <path
              d="M19.6 7.217l-9-6.992c-.4-.3-.9-.3-1.2 0l-9 6.992c-.3.2-.4.5-.4.799v10.987C0 20.701 1.3 22 3 22h14c1.7 0 3-1.299 3-2.997V8.016c0-.3-.1-.6-.4-.8zM12 20.002H8v-7.99h4v7.99zm6-.999c0 .6-.4 1-1 1h-3v-8.99c0-.6-.4-1-1-1H7c-.6 0-1 .4-1 1v8.99H3c-.6 0-1-.4-1-1V8.515l8-6.193 8 6.193v10.488z"
              id="home-svg"
            />
          </defs>
          <use
            fill="currentColor"
            fillRule="nonzero"
            xlinkHref="#home-svg"
          />
        </svg>
      </a>
    </div>
  </div>
</div>
`;

exports[`Storyshots LoadingDots LoadingDots 1`] = `
<div
  className="Loading"
>
  <div
    className="loading-dots"
  />
  <div
    className="loading-dots"
  />
  <div
    className="loading-dots"
  />
</div>
`;

exports[`Storyshots Modal empty modal 1`] = `
<div
  style={
    Object {
      "backgroundColor": "white",
      "boxShadow": "0 2px 16px 0 rgba(0, 0, 0, 0.16)",
      "left": "50%",
      "position": "absolute",
      "top": "50%",
      "transform": "translate(-50%, 0)",
      "zIndex": 1,
    }
  }
>
  <svg
    height={9}
    onClick={[Function]}
    style={
      Object {
        "color": "#191919",
        "cursor": "pointer",
        "float": "right",
        "marginRight": 20,
        "marginTop": 20,
        "opacity": 0.4,
      }
    }
    viewBox="0 0 14 14"
    width={9}
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <defs>
      <path
        d="M13.7 12.3c.4.4.4 1 0 1.4-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3L7 8.4l-5.3 5.3c-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3-.4-.4-.4-1 0-1.4L5.6 7 .3 1.7C-.1 1.3-.1.7.3.3c.4-.4 1-.4 1.4 0L7 5.6 12.3.3c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4L8.4 7l5.3 5.3z"
        id="close-svg"
      />
    </defs>
    <use
      fill="currentColor"
      fillRule="nonzero"
      xlinkHref="#close-svg"
    />
  </svg>
</div>
`;

exports[`Storyshots Modal with a button as children 1`] = `
<div
  style={
    Object {
      "backgroundColor": "white",
      "boxShadow": "0 2px 16px 0 rgba(0, 0, 0, 0.16)",
      "left": "50%",
      "position": "absolute",
      "top": "50%",
      "transform": "translate(-50%, 0)",
      "zIndex": 1,
    }
  }
>
  <svg
    height={9}
    onClick={[Function]}
    style={
      Object {
        "color": "#191919",
        "cursor": "pointer",
        "float": "right",
        "marginRight": 20,
        "marginTop": 20,
        "opacity": 0.4,
      }
    }
    viewBox="0 0 14 14"
    width={9}
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <defs>
      <path
        d="M13.7 12.3c.4.4.4 1 0 1.4-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3L7 8.4l-5.3 5.3c-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3-.4-.4-.4-1 0-1.4L5.6 7 .3 1.7C-.1 1.3-.1.7.3.3c.4-.4 1-.4 1.4 0L7 5.6 12.3.3c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4L8.4 7l5.3 5.3z"
        id="close-svg"
      />
    </defs>
    <use
      fill="currentColor"
      fillRule="nonzero"
      xlinkHref="#close-svg"
    />
  </svg>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <button
      className="Button primary"
      onClick={undefined}
    >
      A button inside a modal
    </button>
  </div>
</div>
`;

exports[`Storyshots Notification notification 1`] = `
<div
  className="Notification"
>
  <div
    id="left-notif"
  >
    <svg
      height={22}
      id="infos-icon"
      viewBox="0 0 22 22"
      width={22}
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M11 0C4.9 0 0 4.9 0 11s4.9 11 11 11 11-4.9 11-11S17.1 0 11 0zm0 20c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm1-9v4c0 .6-.4 1-1 1s-1-.4-1-1v-4c0-.6.4-1 1-1s1 .4 1 1zm-.3-4.7c.2.2.3.4.3.7 0 .3-.1.5-.3.7-.2.2-.4.3-.7.3h-.2c-.1 0-.1 0-.2-.1-.1 0-.1-.1-.2-.1s-.1-.1-.1-.1c-.2-.2-.3-.4-.3-.7 0-.3.1-.5.3-.7l.1-.1c.1 0 .1-.1.2-.1s.1 0 .2-.1c.3 0 .7.1.9.3z"
          id="infos-svg"
        />
      </defs>
      <use
        fill="currentColor"
        fillRule="nonzero"
        xlinkHref="#infos-svg"
      />
    </svg>
    <p>
      some notification
    </p>
  </div>
  <svg
    height={14}
    id="close-icon"
    onClick={undefined}
    viewBox="0 0 14 14"
    width={14}
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <defs>
      <path
        d="M13.7 12.3c.4.4.4 1 0 1.4-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3L7 8.4l-5.3 5.3c-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3-.4-.4-.4-1 0-1.4L5.6 7 .3 1.7C-.1 1.3-.1.7.3.3c.4-.4 1-.4 1.4 0L7 5.6 12.3.3c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4L8.4 7l5.3 5.3z"
        id="close-svg"
      />
    </defs>
    <use
      fill="currentColor"
      fillRule="nonzero"
      xlinkHref="#close-svg"
    />
  </svg>
</div>
`;

exports[`Storyshots QRCode QRCode 1`] = `
<div
  className="QRCode"
  style={
    Object {
      "backgroundImage": "url(http://www.datafakegenerator.com/temp/test409236e0431d1d4dcf66ffba56d9d43c.png)",
    }
  }
/>
`;

exports[`Storyshots SwitchButton primary checked 1`] = `
<div
  className="SwitchButton"
>
  <label
    style={
      Object {
        "color": "blue",
      }
    }
  >
    ENABLED
  </label>
  <div
    className=""
    id="switch-container"
    onClick={[Function]}
  >
    <div
      className=""
      id="background"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className=""
      id="cursor"
      style={
        Object {
          "transform": "translate3d(26px, 0, 0)",
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots SwitchButton primary unchecked 1`] = `
<div
  className="SwitchButton"
>
  <label
    style={
      Object {
        "color": null,
      }
    }
  >
    DISABLED
  </label>
  <div
    className=""
    id="switch-container"
    onClick={[Function]}
  >
    <div
      className=""
      id="background"
      style={Object {}}
    />
    <div
      className=""
      id="cursor"
      style={Object {}}
    />
  </div>
</div>
`;

exports[`Storyshots SwitchButton secondary checked 1`] = `
<div
  className="SwitchButton"
>
  <div
    className="secondary"
    id="switch-container"
    onClick={[Function]}
  >
    <div
      className="secondary"
      id="background"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className="secondary"
      id="cursor"
      style={
        Object {
          "transform": "translate3d(32px, 0, 0)",
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots SwitchButton secondary unchecked 1`] = `
<div
  className="SwitchButton"
>
  <div
    className="secondary"
    id="switch-container"
    onClick={[Function]}
  >
    <div
      className="secondary"
      id="background"
      style={Object {}}
    />
    <div
      className="secondary"
      id="cursor"
      style={Object {}}
    />
  </div>
</div>
`;

exports[`Storyshots Tabs with one tab 1`] = `
<div
  className="Tabs"
>
  <div
    className="active"
    id="tab"
    onClick={[Function]}
  >
    <svg
      height={22}
      viewBox="0 0 16 22"
      width={16}
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M13 0H3C1.3 0 0 1.3 0 3v16c0 1.7 1.3 3 3 3h10c1.7 0 3-1.3 3-3V3c0-1.7-1.3-3-3-3zm1 19c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1V3c0-.6.4-1 1-1h10c.6 0 1 .4 1 1v16zm-6-1a2 2 0 1 0 0-4 2 2 0 0 0 0 4z"
          id="smartphone-svg"
        />
      </defs>
      <use
        fill="currentColor"
        fillRule="nonzero"
        xlinkHref="#smartphone-svg"
      />
    </svg>
    <p>
      DEVICES
    </p>
  </div>
</div>
`;

exports[`Storyshots Tabs with two tabs 1`] = `
<div
  className="Tabs"
>
  <div
    className="active"
    id="tab"
    onClick={[Function]}
  >
    <svg
      height={22}
      viewBox="0 0 16 22"
      width={16}
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M13 0H3C1.3 0 0 1.3 0 3v16c0 1.7 1.3 3 3 3h10c1.7 0 3-1.3 3-3V3c0-1.7-1.3-3-3-3zm1 19c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1V3c0-.6.4-1 1-1h10c.6 0 1 .4 1 1v16zm-6-1a2 2 0 1 0 0-4 2 2 0 0 0 0 4z"
          id="smartphone-svg"
        />
      </defs>
      <use
        fill="currentColor"
        fillRule="nonzero"
        xlinkHref="#smartphone-svg"
      />
    </svg>
    <p>
      DEVICES
    </p>
  </div>
  <div
    className="inactive"
    id="tab"
    onClick={[Function]}
  >
    <svg
      height={20}
      viewBox="0 0 20 20"
      width={20}
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M17 0H3C1.3 0 0 1.3 0 3v14c0 1.7 1.3 3 3 3h14c1.7 0 3-1.3 3-3V3c0-1.7-1.3-3-3-3zM3 2h14c.6 0 1 .4 1 1v3H2V3c0-.6.4-1 1-1zM2 17V8h4v10H3c-.6 0-1-.4-1-1zm15 1H8V8h10v9c0 .6-.4 1-1 1z"
          id="apps-svg"
        />
      </defs>
      <use
        fill="currentColor"
        fillRule="nonzero"
        xlinkHref="#apps-svg"
      />
    </svg>
    <p>
      APPLICATIONS
    </p>
  </div>
</div>
`;

exports[`Storyshots TextInput with error 1`] = `
<div
  className="TextInput error"
>
  <input
    defaultValue={undefined}
    onChange={undefined}
    type="text"
  />
  <label>
    some button
  </label>
</div>
`;

exports[`Storyshots TextInput with one label 1`] = `
<div
  className="TextInput"
>
  <input
    defaultValue={undefined}
    onChange={undefined}
    type="text"
  />
  <label>
    some input
  </label>
</div>
`;

exports[`Storyshots TextInput with one label and a default value 1`] = `
<div
  className="TextInput"
>
  <input
    defaultValue="default text"
    onChange={undefined}
    type="text"
  />
  <label>
    some button
  </label>
</div>
`;
